src/guides/v2.3/comp-mgr/patching.md

```diff
@@ -75,14 +79,164 @@ index c8a6fef58d31..7d01c195791e 100644
 
 ## Applying patches
 
-There are two ways to apply patches:
+There are three ways to apply patches:
```

Comment:
Does it matter that there are three.  Consider:  You can use any of the following methods to apply patches:

Reply:
I suppose not. I was just going with what was already there.